setup.cfg

```
@@ -36,7 +36,7 @@ install_requires =
     niworkflows >= 1.7.0
     nitransforms >= 23.0.1
     numpy >= 1.21.0
-    pybids >= 0.15.1
+    pybids >= 0.15.1, <= 0.16.1
```

Maximums are painful. Can I suggest a `!=` to allow for bug fixes?

```suggestion
    pybids >= 0.15.1, != 0.16.1
```